public/index.html

```diff
@@ -0,0 +1,152 @@
+              <p style="margin-bottom:1.6vw;">
+                When taking a picture, line<br>up the edge with the<br>dotted line on your card.
+              </p>
+              <p style="margin-bottom:1.6vw;">
+                Your images should be well<br> lit and look like this <br>example.
+              </p>
+            </div>
+            <div><span class="learn">Learn more <i class="fas fa-angle-double-right"></i></span></div>
+          </div>
+        </div>
+        <div class="warnings">
+          <h5>Your data will not be saved or uploaded when you select an image.</h5>
+        </div>
+      </div>
+
+    </div>
+    <div class="screen container-3">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">2. Now click in the center of the left patch.</h3>
+            <div id="gallery" class="input-img-op">
+
+            </div>
+        </div>
+        <div class="badge-img">
+          <img src="./images/colorileft.png" alt="">
+        </div>
+        <div class="warnings">
+          <h5>Your selection should look like this. <span class="learn">Learn more <i
+                class="fas fa-angle-double-right"></i></span></h5>
+        </div>
+        <div id="loading-1" class="hide"></div>
+        <div class="buttons-div">
+          <button type="button" class="btn btn-back" onclick="changeScreen(event)"><i class="fa fa-angle-double-left"
+              aria-hidden="true"></i> Go Back</button>
+          <button type="button" class="btn btn-next"  onclick="changeScreen(event)">Next </i></button>
+        </div>
+      </div>
+
+    </div>
+    <div class="screen container-4">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">3. Now click in the center of the right patch.</h3>
+            <div id="stepTwo" class="input-img-op">
+
+            </div>
+        </div>
+        <div class="badge-img">
+          <img src="./images/coloriright.png" alt="">
+        </div>
+        <div class="warnings">
+          <h5>Your selection should look like this. <span class="learn">Learn more <i
+                class="fas fa-angle-double-right"></i></span></h5>
+        </div>
+        <div id="loading-2" class="hide"></div>
+        <div class="buttons-div">
+          <button type="button" class="btn btn-back" onclick="changeScreen(event)"><i class="fa fa-angle-double-left"
+              aria-hidden="true"></i> Go Back</button>
+          <button type="button" class="btn btn-next"  onclick="changeScreen(event)">Next </i></button>
+        </div>
+      </div>
+
+    </div>
+    <div class="screen container-5">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">4. Results</h3>
+            <div id="final-res" class="input-img-op">
+
+            </div>
+        </div>
+        <div class="result">
+          <h4>The RIGHT side is</h4>
+          <h2 class="res-num">15 % DARKER</h2>
```

Comment:
Are you sure this percentage should be hard coded?

Reply:
Not needed now!
Thanks!